Gaijinizer patch/Map119.txt

```diff
@@ -263,9 +272,10 @@ incarnated to such a demon.#MTLed
  織物を司る女神が、普通の人間と勝負して負けるだなんて
  あっちゃいけないからねぇ。
 > CONTEXT: Map119/events/5/pages/1/55/Dialogue/enemy10
-...... This is a long time ago story.
-The goddess who controls the fabric can defeat an
-ordinary human and lose. I can't wait.#MTLed
+Kuku... This is a story from a long time ago.
+
+The story of the goddess of textiles being
+defeated by a mere human.#RonIffy
 > END STRING
```

Comment:
Is this a valid interpretation, keeping in with text?

>_**The story of a goddess being defeated by
the weaving of a mere human.**_

Reply:
That works well.